README.md

```diff
@@ -64,6 +64,7 @@ React Firebase Admin is our in-house admin dashboard boilerplate, used in many o
 - PWA ready thanks to CRA and Firebase
 - Multi-tenancy
 - Internationalization (English/Spanish)
+- Interface to choose between Real Time Database and Firestore
```

Comment:
Maybe we should change this to: "Ability to choose between realtime database or firestore"